src/components/internshipsFilter/InternshipsFilter.jsx

```diff
@@ -225,9 +243,14 @@ export const InternshipsFilter = ({ onFilter }) => {
               ))}
             </Select>
           </FormControl>
-          <Button onClick={handleSubmit} size="small" variant="contained">
-            {getFieldLabel('common.filter')}
-          </Button>
+          <Box display="flex" justifyContent="space-between">
+            <Button onClick={handleSubmit} size="small" variant="contained">
+              {getFieldLabel('common.filter')}
+            </Button>
+            <Button onClick={cleanFilter} size="small" variant="contained">
+              {getFieldLabel('common.clean.filter')}
```

Comment:
I think would be better to name just Clean or Reset.  If you think otherwise, then leave it as it is

Reply:
I think Reset will be better